### DISCUSSION TITLE
Why no recent version (discourages potential users)?

### DISCUSSION THREAD

**Discussion by zzapper** [autokey/autokey#679] [General]
Hi
 I love and promote autokey but there hasn't been a release since Feb 2020 V0.95.10 .

If you are a potential user you might worry that AK is a dormant project and carry on searching.

I have no need of any improvements but I know other projects cover this issue by doing a so called patch or security release.

I know you are at Beta 10 but you have to dig down a little to see that!

Sorry hope this doesn't annoy anyone I mean well!

(Anyway great little app)
zzapper

**Comment by josephj11** [comment]
This is a valid issue. (Soapbox warning! LOL)

It's not for lack of trying though. We currently have one main developer who was gracious enough to assume that role (when our former developer ghosted us) - even though he really didn't want to assume that responsibility. He's a hard science PhD candidate which means he has quite a bit else on his plate. We have one more frequent contributor who hasn't been seen much lately and there are a few others who have contributed small PRs. Basically, we're just critically understaffed.

We are hoping to release the Beta as 0.96.0 soon, but we could really use some more users with different environments to test it some more. AFAIK, only a small handful of users are using it so far and we really don't want to get hit by an onslaught of bug reports and problems if we release without adequate testing - to say nothing of what that might do to our reputation!

0.96.0 has a number of great new features and a lot of fixes in it. It needs to get out. There are also a number of other features - a few of which I really want that are either in the development repo or still just enhancement requests and none of them can get done until after 0.96.0 or maybe even 0.96.1 comes  out.

As much as possible, shill AutoKey on places like stackexchange (by answering relevant questions and upvoting AutoKey related questions and answers  - if they deserve it). There's an autokey tag in several of the Q&As you can subscribe to.  Eventually, maybe some competent devs will be inspired to give us a hand. The devs we have now are great, but they really need help to get more done.

The original branch of AutoKey died from developer burnout. Fortunately, this branch got started before that happened. Having even one continuously active developer is not something to take for granted!

AutoKey is complex enough that continuity of developers really matters. It also relies on a number of sophisticated upstream components for which we have never had deep expertise. In particular, getting AutoKey to work with Wayland is by no means assured.

**Reply by zzapper** [reply]
Josephj11 Thanks for this detailed reply I appreciate the situation better now (we all overload today). I guess a premature release which say broke a lot of existing scripts would cause a lot of upset!. Just to re-mention that some projects fudge this issue by doing a so called 'security or patch release' which has no or few new features.
